dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -97,14 +106,30 @@ def debug_str(self):
 
         return s
 
+    def check_import(self, lib_path):
+        import tensorflow as tf
+        assert can_import_dali()  # DALI symbols need to be loaded
```

Comment:
Assert or just false?
We just want to fail if there is no exact match in this case or build from source anyway?

Reply:
The idea is that you only call check_import if you can import the plugin. It should only be called if can_import_dali()==True. I'll correct the logic